src/mixins/allowable-actions-mixin.ts

```diff
@@ -35,7 +35,8 @@ export default class AllowableActionsMixin extends Vue {
       }
 
       case AllowableActions.DOWNLOAD_BUSINESS_SUMMARY: {
-        return (!!businessId && !!getFeatureFlag('download-summary-enabled'))
```

Comment:
Please add a comment to the release ticket to delete this FF from LD once this project is deployed to Prod.

Reply:
Will do!